src/main/java/xreliquary/items/ItemLanternOfParanoia.java

```diff
@@ -68,16 +71,25 @@ public void onUpdate(ItemStack stack, World world, Entity e, int i, boolean f) {
 			int playerY = MathHelper.floor(player.getEntityBoundingBox().minY);
 			int playerZ = MathHelper.floor(player.posZ);
 
+			BlockPos pos;
```

Comment:
These variables should be defined closer to their usage

Reply:
My bad, I've a bunch of programming languages floating around in my head and for some reason I thought this was the most efficient way to handle things without instantiating a number of extra variables per iteration.